.ci/scripts/build_all_plugins.sh

```diff
@@ -3,6 +3,11 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+arg="$1"
 cd all_packages
-flutter build windows --debug
-flutter build windows --release
+if [[ "$arg" == "web" ]]; then
+  echo "Skipping; web does not support debug builds"
+else
+  flutter build "$arg" --debug
+  flutter build "$arg" --release
```

Comment:
Although relatedly, I didn't consider this before: in Cirrus I have this as two different script steps because it means that the output for each part is clearly separated in the results UI. That's *much* more usable than having to scroll through a bunch of output with multiple runs and the failure somewhere in the middle.

I think for LUCI the best option would be to make this take two args, the platform and the build mode, then have the yaml file call it twice, each from a separate task. Then the web logic doesn't need to be here at all, it would just be a comment in a future `web_build_all_plugins.yaml` about why it's not doing debug like the other platforms.

Reply:
Makes sense to me. Updated to accept those two args.